ocs_authentication/auth_profile/migrations/0001_initial.py

```diff
@@ -10,18 +9,16 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
This was autogenerated with the django makemigrations, but was giving me an issue in testing and did not seem necessary as a dependency so I removed it.